testplan/testing/base.py

```diff
@@ -195,7 +195,11 @@ def report(self):
     @property
     def stdout_style(self):
         """Stdout style input."""
-        return self.cfg.stdout_style
+        return (
+            self.cfg.stdout_style
+            if hasattr(self.cfg, "stdout_style")
```

Comment:
can stdout_style have a default value, so that we don't need to do hasattr check?